### PR TITLE
Added IKE counters

### PR DESCRIPTION
**Pull request by yuri-zubov**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [x] Update Changelog following the conventions laid out on [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass

#### New Plugins

- [x] Tests

- [ ] Add the plugin to the README

- [x] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +26,6 @@ deploy:
   on:
     tags: true
     all_branches: true
-    rvm: 2.0
```
Comment (mbbroberg) [inline .travis.yml:30]: Thank you for removing 2.0! That does mean we need a little more in the CHANGLOG. Could you add the following, also under `Unreleased`: 

```
### Breaking Changes
Dropping ruby `< 2.1` support (@yuri-zubov)
```

**Review by mbbroberg — APPROVED** [review]
❤️Awesome addition to the plugin! I'll add the minor changelog edit needed. Holding off on merging only because I want to ensure we cut a release right after and I'm not familiar with that process just yet 😄

**Review by mbbroberg — CHANGES_REQUESTED** [review]
Actually I can't edit your PR - please add that changelog item and then we're 👍. I'd love a test or two if you have time to write them, but we won't gate the PR because of it. More on testing: 

See [this plugin](https://github.com/sensu-plugins/sensu-plugins-mesos/tree/master/test) for an example of solid test coverage. Also note we no longer use Code Climate so please ignore (or remove) it in `spec_helper.rb`. Thanks!

**Comment by yuri-zubov** [comment]
@mbbroberg recheck again, please

**Review by mbbroberg — APPROVED** [review]
Perfection 👍. Really excited to get these merged and released!!

**Comment by yuri-zubov** [comment]
@mbbroberg I have just fixed gramma (founded to found)

File: .travis.yml (hunk 1 above)
Comment (majormoses) [reply .travis.yml:30]: technically this is just removing the testing, not removing support, the `.gemspec` needs to be updated so I will do that prior to release.

**Comment by majormoses** [comment]
travis is having issues if it does not auto release eventually I will do it manually later. Ping me if I don't reply back in 24 hours.

**Comment by majormoses** [comment]
looks like it released: https://rubygems.org/gems/sensu-plugins-strongswan/versions/1.2.0

**Pull request merged**